.bazelci/presubmit.yml

```diff
@@ -1,5 +1,8 @@
 ---
 validate_config: 1
+buildifier:
+  version: latest
+  warnings: "all"
```

Comment:
How widely accepted is buildifier?

Reply:
It's the officially recommended way to format (&edit) your `BUILD` files, so I'd say it's widely accepted.

https://github.com/bazelbuild/buildtools/tree/master/buildifier

Reply:
OK. (This looks official enough: https://docs.bazel.build/versions/master/skylark/build-style.html)